exonum-java-binding/doc/Migration_guide_0.6.md

```diff
@@ -0,0 +1,165 @@
+-        <configuration>
+-          <outputFile>${project.build.directory}/cryptocurrency-classpath.txt</outputFile>
+-          <includeScope>runtime</includeScope>
+-        </configuration>
++        <artifactId>maven-assembly-plugin</artifactId>
+         <executions>
+           <execution>
+-            <id>generate-classpath-file</id>
++            <id>package-service-artifact</id>
++            <phase>package</phase>
+             <goals>
+-              <goal>build-classpath</goal>
++              <goal>single</goal>
+             </goals>
+           </execution>
+         </executions>
++        <configuration>
++          <descriptorRefs>
++            <descriptorRef>jar-with-dependencies</descriptorRef>
++          </descriptorRefs>
++          <finalName>${project.artifactId}-${project.version}-artifact</finalName>
++          <appendAssemblyId>false</appendAssemblyId>
++          <archive>
++            <manifestEntries>
++              <Plugin-Id>${project.groupId}:${project.artifactId}:${project.version}</Plugin-Id>
++              <Plugin-Version>${project.version}</Plugin-Version>
++              <Plugin-Provider>${project.groupId}</Plugin-Provider>
++            </manifestEntries>
++          </archive>
++        </configuration>
+       </plugin>
+ 
+       <!-- Skip the deployment of internal module as it is inherited from parent pom -->
+Index: exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ServiceModule.java
+===================================================================
+--- exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ServiceModule.java	(date 1552387521000)
++++ exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/CryptocurrencyServiceModule.java	(date 1552405034000)
+@@ -17,13 +17,14 @@
+ package com.exonum.binding.cryptocurrency;
+ 
+ import com.exonum.binding.cryptocurrency.transactions.CryptocurrencyTransactionConverter;
++import com.exonum.binding.service.AbstractServiceModule;
+ import com.exonum.binding.service.Service;
+ import com.exonum.binding.service.TransactionConverter;
+-import com.google.inject.AbstractModule;
+ import com.google.inject.Singleton;
++import org.pf4j.Extension;
+ 
+-@SuppressWarnings("unused") // Instantiated through reflection by Guice.
+-public final class ServiceModule extends AbstractModule {
++@Extension
++public final class CryptocurrencyServiceModule extends AbstractServiceModule {
+ 
+   @Override
+   protected void configure() 
+
+```
+
+</details>
+
+## Update the node configuration
+
+Below are the highlights of the changes in node configuration:
+
+1. It is no longer needed to pass the system nor service classpath.
+   Paths to service artifacts are specified in "ejb_app_services.toml" 
+   configuration file in the *working directory* of each node application.
+1. Some configuration options were moved between configuration phases.
```

Comment:
I think it is not worth documenting each changed parameter, as they are listed in the tutorial for each phase. Is a single link to it enough?